src/plans/solver_state.jl

```diff
@@ -278,14 +361,65 @@ function update_storage!(a::AbstractStateAction, s::AbstractManoptSolverState)
     return a.keys
 end
 
+function _storage_key_true(nt::NamedTuple)
```

Comment:
What does this function do? For me the name is mega-cryptic. Does it check whether a key exists? But why does it then only have one parameter?

A small doc and maybe a better function name would help, I am currently not able to understand what it is for and how it is used.

Reply:
Hm... when I was writing an answer for you I've noticed that it can be done in a much more simple way. I will change it.

Reply:
even better – now the code looks fine – I do not yet fully understand it, but it resolved this.